<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>iGoBeatTools</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Aerial</family>
        <pointsize>44</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>iGoBeatPro Tools</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="openExternalLinks">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSlideNode"/>
   <addaction name="action3In1"/>
   <addaction name="actionBmse_to_iGoBeat"/>
   <addaction name="actionPackageSongs"/>
   <addaction name="actionPointTranslate"/>
   <addaction name="actionAddHd"/>
   <addaction name="actionScalePng"/>
   <addaction name="actionEncryption"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>726</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionPointTranslate"/>
    <addaction name="actionEncryption"/>
    <addaction name="actionPackageSongs"/>
    <addaction name="actionAddHd"/>
    <addaction name="actionScalePng"/>
    <addaction name="actionBmse_to_iGoBeat"/>
    <addaction name="action3In1"/>
    <addaction name="actionSlideNode"/>
   </widget>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionPointTranslate">
   <property name="text">
    <string>ipad坐标转iphone</string>
   </property>
  </action>
  <action name="actionPackageSongs">
   <property name="text">
    <string>歌曲打包</string>
   </property>
  </action>
  <action name="actionAddHd">
   <property name="text">
    <string>图片增加hd后缀</string>
   </property>
  </action>
  <action name="actionScalePng">
   <property name="text">
    <string>缩放图片</string>
   </property>
  </action>
  <action name="actionSlideNode">
   <property name="text">
    <string>滑动音符编号</string>
   </property>
  </action>
  <action name="actionBmse_to_iGoBeat">
   <property name="text">
    <string>bmse曲谱转换</string>
   </property>
  </action>
  <action name="actionEncryption">
   <property name="text">
    <string>加密</string>
   </property>
  </action>
  <action name="action3In1">
   <property name="text">
    <string>zip打包编号三合一</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
